src/resonanceReconstruction/rmatrix/src/makeQuantumNumbers.hpp

```diff
@@ -19,3 +19,53 @@ makeQuantumNumbers( const ParticlePair& incident, unsigned int lmax ) {
   }
   return numbers;
 }
+
+ChannelQuantumNumbers
+retrieveQuantumNumber( unsigned int l, double j,
+                       std::vector< ChannelQuantumNumbers >& available ) {
+
+  // useful lambdas
+  auto compareNumbers = [] ( const auto& left, const auto& right ) -> bool {
+
+    return
+    ( left.orbitalAngularMomentum() == right.orbitalAngularMomentum() ) and
```

Comment:
At some point I was asked to write `and` rather than `&&`, etc., in GNDStk. Later I spoke with a guy I know who's on the C++ standards committee, who wrote back with a detailed explanation of why the original terminology - `&&`, `||` etc. should be preferred. At some point I think we should all have a discussion about this. I personally agree that we should use the original C++ terminology.